Sources/Danger-Swift/Fake.swift

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!--
  0 failure: 
  1 warning:  Files should have...
  
  
  DangerID: danger-id-default;
  File: Sources/Danger-Swift/Fake.swift;
  Line: 1;
-->

- :warning: Files should have a single trailing newline. (`trailing_newline`)